services/clinical/index.js

```diff
@@ -63,6 +63,17 @@ const commitRegistrationData = async (programShortName, registrationId, Authoriz
   return response;
 };
 
+const getClinicalSubmissionData = async (programShortName, Authorization) => {
+  const url = `${CLINICAL_SERVICE_ROOT}/submission/program/${programShortName}/clinical/upload`;
```

Comment:
just a note, generally would have suggested using `url-join` for url construction, but as long as it works :)